src/annotator/components/AdderToolbar.tsx

```diff
@@ -112,25 +113,26 @@ function ToolbarButton({ badgeCount, Icon, label, onClick, shortcut }) {
   );
 }
 
-/**
- * Union of possible toolbar commands.
- *
- * @typedef {'annotate'|'highlight'|'show'|'hide'} Command
- */
+export type Command = 'annotate' | 'highlight' | 'show' | 'hide';
```

Comment:
This type is used by `adder`. Do we need to establish a convention about how we want to handle type exports?

Reply:
If you want to use the type in other modules, or intend for it to be directly used (eg. an interface that we expect classes to declare they implement), then it needs to be exported. We haven't yet established conventions around types that _could_ be used, but aren't often (eg. props types). One possible policy is to export all types which are referenced in the module's value exports (functions, classes, variables).